src/Bridge/Symfony/SonataDoctrineBundle.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Doctrine\Bridge\Symfony;
+
+use Sonata\Doctrine\Bridge\Symfony\DependencyInjection\Compiler\AdapterCompilerPass;
+use Sonata\Doctrine\Bridge\Symfony\DependencyInjection\Compiler\MapperCompilerPass;
+use Sonata\Doctrine\Bridge\Symfony\DependencyInjection\SonataDoctrineExtension;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+final class SonataDoctrineBundle extends Bundle
+{
+    public function build(ContainerBuilder $container)
+    {
+        $container->addCompilerPass(new AdapterCompilerPass());
+        $container->addCompilerPass(new MapperCompilerPass());
+    }
+
+    protected function getContainerExtensionClass()
+    {
+        return SonataDoctrineExtension::class;
```

Comment:
Why is this needed, Symfony cant get this class name right? maybe it was because the old structure and now it works, can you try?

Reply:
It's working. Thanks.